### PR TITLE
feat: Use death count in autoshutdown.

### PR DESCRIPTION
**Pull request by laxect**
This patch uses a death count to decide whether the server needs auto shutdown or not.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -198,23 +198,31 @@ def cancel_shutdown(update: Update, context: CallbackContext):
...
-        context.bot.send_message(CHAT, f"过久没人在线，{wait_time_min}分钟后关闭服务器，若要游玩请重新打开。\n发送 /cancel_shutdown@{context.bot.username} 取消关机。")
-        os.system(f'shutdown -P +{wait_time_min}')
+    DEATH_COUNT = 'DEATH_COUNT'
+    death_count = context.bot_data.get(DEATH_COUNT, 0)
+    wait_time_min = 5
 
     try: 
         online = command('list')
+        matched = re.search(r'\d+', online) 
+        last_online = int(matched.group(0))
```
Comment (uonr) [inline bot.py]: Should be `current_online`

File: bot.py [hunk 2]
```diff
@@ -271,7 +279,7 @@ def main():
     dispatcher.add_handler(MessageHandler(Filters.status_update.new_chat_title, status_update))
     dispatcher.add_handler(MessageHandler(Filters.text & ~Filters.command, forward_to_minecraft))
 
-    dispatcher.job_queue.run_repeating(auto_shutdown, interval=1 * 60 * 60, first=0)
+    dispatcher.job_queue.run_repeating(auto_shutdown, interval= 60, first=0)
```
Comment (uonr) [inline bot.py]:
```suggestion
    dispatcher.job_queue.run_repeating(auto_shutdown, interval=60, first=0)
```

File: bot.py (hunk 1 above)
Comment (laxect) [reply bot.py]: Fixed!

File: bot.py (hunk 2 above)
Comment (laxect) [reply bot.py]: Fixed! Thanks for review.

**Pull request merged**